elizaOS/eliza
# bugfix. the port 80 is not listening use 3000

3000 is listening on the docker image

# Risks

Low - This change affects the Docker container configuration. It may require updates to documentation and environment configurations where the container is deployed.

---

# Background

## What does this PR do?
This pull request updates the Dockerfile and related Docker configuration to expose port 3000 instead of port 80. This change is necessary to align with our application's configuration and avoid conflicts with other services running on port 80.

## What kind of change is this?

Bug fixes (non-breaking change which fixes an issue)


N/A (Issue linked provides context)

---

# Documentation changes needed?

My changes require a change to the project documentation. I have updated the documentation accordingly.

---

# Testing

## Where should a reviewer start?
Start by reviewing the changes in the Dockerfile and docker-compose.yml files.

## Detailed testing steps

1. Pull the latest changes from the repository.
2. Build the Docker image using the updated Dockerfile.
3. Run the container and verify that it is listening on port 3000.
4. Access the application through port 3000 to ensure it is functioning correctly.



---



N/A



N/A



N/A



docker build


## Discord username m1337d
